pkg/test/resourcefixture/testdata/basic/essentialcontacts/v1alpha1/essentialcontactscontact/essentialcontactscontact-minimal/create.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: essentialcontacts.cnrm.cloud.google.com/v1alpha1
+kind: EssentialContactsContact
+metadata:
+  name: essentialcontactscontact-minimal-${uniqueId}
+spec:
+  projectRef:
+    external: ${projectId}
+  languageTag: en
+  notificationCategorySubscriptions:
+    - BILLING
+  email: "bill-alerts@test.com"
```

Comment:
Suggest changing this to `IAMServiceAccountRef` https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/apis/refs/v1beta1/gcpserviceaccountref.go

Reply:
This is just an email. Not a IAM Service Account. https://cloud.google.com/resource-manager/docs/managing-notification-contacts#essential-contacts-add-console